scripts/bootstrap.bat

```diff
@@ -223,7 +224,7 @@ echo %date% %time% ======================================================
 echo %date% %time% Setting Python path
 echo %date% %time% ======================================================
 echo.
-set PYTHONPATH=.;%cd%\pluginExamples;%cd%\..\Wasatch.PY;%CONDA_PREFIX%\lib\site-packages
+set PYTHONPATH=.;%cd%\pluginExamples;%cd%\..\Wasatch.PY;%cd%\enlighten\assets\uic_qrc;%CONDA_PREFIX%\lib\site-packages
```

Comment:
This was preventing the installed enlighten.exe from finding package grey_icons_rc (imported by enlighten_layout.py).  I don't know why this didn't come up earlier.